addons/website_sale/static/src/js/website_sale_utils.js

```diff
@@ -20,14 +20,15 @@ function getNavBarButton(selector) {
 }
 
 function animateClone($cart, $elem, offsetTop, offsetLeft) {
-    $cart.find('.o_animate_blink').addClass('o_red_highlight o_shadow_animation').delay(500).queue(function(){
+    $cart.find('.o_animate_blink').addClass('o_red_highlight o_shadow_animation').delay(500).queue(function () {
         $(this).removeClass("o_shadow_animation").dequeue();
-    }).delay(2000).queue(function(){
+    }).delay(2000).queue(function () {
         $(this).removeClass("o_red_highlight").dequeue();
     });
     var $imgtodrag = $elem.find('img').eq(0);
     if ($imgtodrag.length) {
         var $imgclone = $imgtodrag.clone()
+            .css('z-index', 1050) // Get over header which is 1030
```

Comment:
Should not be in JS, what if we change the value in css ?

Reply:
Is there a scss class already in place for that or should i create a new one for that ?

Reply:
The class which is on the animated element or that is added on the animated element